syntax = "proto3";
package Protocol;

import "game_enum.proto";
import "game_struct.proto";

message GameClientLogin
{
	int64 auth = 1;
}

message GameServerLogin
{
	int64 auth = 1;
}

message GameServerLoadCharacter
{
	kLoadState state = 1;
	int64 account_id = 2;
	int64 character_id = 3;
	bytes name = 4;
	bool gender = 5;
	int32 face_id = 6;
	int32 str = 7;
	int32 dex = 8;
	int32 int = 9;
	int32 luk = 10;

	repeated item items = 11;
}
